.jenkins/jenkins.sh

```diff
@@ -133,6 +133,11 @@ if [ ${container_engine} == "sarus" ]; then
     fi
 fi
 
+if [ ${host} == "daint" ]; then
+    daintenv=${SCRATCH}/vcm_env_${BUILD_TAG}
+    ${root}/../install_virtualenv.sh ${daintenv}
```

Comment:
why is this parent of root?

Reply:
I was thinking it would be useful to have a script to install a fv3core virtualenv at the top level. It doesn't quite work currently because you still need to have a few other things installed, like gt4py, and other things that happen in the daint specific script. I am ok with putting it in the jenkins directory for now since it won't be used outside of daint yet.

Reply:
either way is ok with me